spec/jobs/handover_follow_up_job_spec.rb

```diff
@@ -0,0 +1,186 @@
+  end
+
+  it 'does not send emails for offenders that do not exist' do
+    offender = build_offender(sentence_type: :determinate, ard_crd_release: nil, ted: nil)
+    create(:case_information, nomis_offender_id: offender.offender_no,
+                       team: build(:team, local_divisional_unit: build(:local_divisional_unit)))
+
+    create(:allocation, nomis_offender_id: offender.offender_no)
+
+    allow(OffenderService).to receive(:get_offender).and_return(nil)
+
+    expect_any_instance_of(CommunityMailer).not_to receive(:urgent_pipeline_to_community).and_call_original
+    expect { described_class.perform_now(Time.zone.today) }.not_to raise_error
+  end
+
+  it 'does not send emails for offenders who have a COM assigned' do
+    case_info = create(:case_information, nomis_offender_id: determinate_offender.offender_no, com_name: "Betty White")
+    create(:allocation, nomis_offender_id: determinate_offender.offender_no)
+
+    allow(OffenderService).to receive(:get_offender).and_return(determinate_offender)
+    determinate_offender.load_case_information(case_info)
+
+    expect_any_instance_of(CommunityMailer).not_to receive(:urgent_pipeline_to_community).and_call_original
+    described_class.perform_now(Time.zone.today)
+  end
+
+  it 'does not send emails for offenders who do not have a handover_start_date' do
+    indeterminate_without_ted = build_offender(Time.zone.today + 10.months,
+                                               sentence_type: :indeterminate,
+                                               ard_crd_release: nil,
+                                               ted: nil)
+
+    case_info = create(:case_information, nomis_offender_id: indeterminate_without_ted.offender_no,
+                       team: build(:team, local_divisional_unit: build(:local_divisional_unit)))
+
+    create(:allocation, nomis_offender_id: indeterminate_without_ted.offender_no)
+
+    allow(OffenderService).to receive(:get_offender).and_return(indeterminate_without_ted)
+    indeterminate_without_ted.load_case_information(case_info)
+
+    expect_any_instance_of(CommunityMailer).not_to receive(:urgent_pipeline_to_community).and_call_original
+    described_class.perform_now(Time.zone.today)
+  end
+
+  context 'when no COM assigned' do
+    let(:case_info) do
+      create(:case_information, nomis_offender_id: determinate_offender.offender_no,
+                       team: build(:team, local_divisional_unit: build(:local_divisional_unit)))
+    end
+
+    before do
+      allow(OffenderService).to receive(:get_offender).and_return(determinate_offender)
+    end
+
+    it 'does not send emails for offenders whose are not yet approaching handover' do
+      create(:allocation, nomis_offender_id: determinate_offender.offender_no)
+      determinate_offender.load_case_information(case_info)
+
+      expect_any_instance_of(CommunityMailer).not_to receive(:urgent_pipeline_to_community).and_call_original
```

Comment:
the 'and_call_original' doesn't make much sense here as we're expecting not to be called

Reply:
Can you suggest what I should be doing then?